iosApp/iosApp/Pages/TripDetails/VehicleCardView.swift

```diff
@@ -35,24 +33,15 @@ struct VehicleCardView: View {
 struct VehicleOnTripView: View {
     let vehicle: Vehicle
     let route: Route
-    let line: Line?
     let stop: Stop
     let trip: Trip
 
     var backgroundColor: Color {
-        if route.id.starts(with: "Shuttle"), let line {
```

Comment:
this was the only other place we were checking for Shuttles to resolve color, so figured I'd clean it up too